workers/ohsome_quality_analyst/indicators/mapping_saturation/indicator.py

```diff
@@ -35,162 +45,183 @@ def __init__(
             layer_name=layer_name,
             feature=feature,
         )
-        self.time_range = time_range
+        self.time_range: str = time_range
+
         # The following attributes will be set during the life-cycle of the object.
         # Attributes needed for calculation
         self.values: list = []
-        self.values_normalized: list = []
+        self.latest_value: Union[None, int, float] = None
         self.timestamps: list = []
-        self.no_data: bool = False
-        self.deleted_data: bool = False
+
+        self.upper_threshold = 0.97  # Threshold derived from Gröchenig et al.
+        # TODO: What is a good lower threshold?
+        self.lower_threshold = 0.30
 
         # Attributes needed for result determination
-        self.saturation = None
-        self.growth = None
+        self.best_fit: Optional[models.BaseStatModel] = None
+        self.fitted_models: List[models.BaseStatModel] = []
 
     async def preprocess(self) -> None:
         query_results = await ohsome_client.query(
-            layer=self.layer, bpolys=self.feature.geometry, time=self.time_range
+            layer=self.layer,
+            bpolys=self.feature.geometry,
+            time=self.time_range,
         )
-        self.values = [item["value"] for item in query_results["result"]]
-        self.timestamps = [
-            isoparse(item["timestamp"]) for item in query_results["result"]
-        ]
-        # Latest timestamp of ohsome API results
-        self.result.timestamp_osm = self.timestamps[-1]
-        max_value = max(self.values)
-        if max_value == 0:
-            self.no_data = True
-        elif self.values[-1] == 0:
-            self.deleted_data = True
-        else:
-            self.values_normalized = [value / max_value for value in self.values]
+        for item in query_results["result"]:
+            self.values.append(item["value"])
+            self.timestamps.append(isoparse(item["timestamp"]))
+        self.latest_value = self.values[-1]
 
     def calculate(self) -> None:
-        """Calculate the growth rate and saturation level within the last 3 years."""
-        if self.no_data:
-            self.result.description = "No features were mapped in this region."
+        # Latest timestamp of ohsome API results
+        self.result.timestamp_osm = self.timestamps[-1]
```

Comment:
General question: where is documented what `result.timestamp_osm` is? Shouldn't this be in the documentation or the base indicator?

Reply:
True. Thats the place where it should be documented. 
I added some documentation to the base Result class